crates/fuel-core/src/schema/chain.rs

```diff
@@ -280,7 +280,9 @@ impl GasCosts {
             GasCostsValues::V1(_)
             | GasCostsValues::V2(_)
             | GasCostsValues::V3(_)
-            | GasCostsValues::V4(_) => GasCostsVersion::V1,
+            | GasCostsValues::V4(_)
+            //TODO: Handle new GasCostsValues variant properly
+            | GasCostsValues::V5(_) => GasCostsVersion::V1,
```

Comment:
@AurelienFT see if this is okay, otherwise can be handled on a PR on top of this

Reply:
I think we have to create a `GasCostsVersion::V2` but it's two lines Ill add it there